src/test/kotlin/org/rust/ide/refactoring/RsExtractEnumVariantTest.kt

```diff
@@ -42,7 +42,11 @@ class RsExtractEnumVariantTest : RsTestBase() {
             V2
         }
     """, """
-        struct /*caret*/V1 { pub a: i32, pub b: bool, pub c: String }
+        struct /*caret*/V1 {
+            pub a: i32,
+            pub b: bool,
+            pub c: String
```

Comment:
Hmm, where is trailing comma here? Is it the same issue that `RsTrailingCommaFormatProcessor` doesn't invoke for some reason?

Reply:
Yes, I tried it now and the comma processor doesn't get invoked.

Reply:
Oh